project/plugins.sbt

```diff
@@ -1,2 +1,2 @@
 
-addSbtPlugin("io.verizon.build" % "sbt-rig" % "1.1.20")
+addSbtPlugin("io.verizon.build" % "sbt-rig" % "1.2.25")
```

Comment:
Needs to be version `1.2.26` as that is the one that I just published with your other contribution to sbt-rig :-)